pkgs/by-name/eo/eon/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "eon";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "emilk";
+    repo = "eon";
+    tag = finalAttrs.version;
+    hash = "sha256-dsm4r3Or1QDA32VqERnak01vHcf7E8d/+F/S+lmMedA=";
+  };
+
+  cargoHash = "sha256-oaYykh2NtGPlDq5JSuCMaKmD65b1xZbeuGUDZD/STD0=";
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Simple and friendly config format";
+    longDescription = ''
+      The simple and friendly config format aimed to be a replacement
+      for Toml and Yaml.
+    '';
+    homepage = "https://github.com/emilk/eon";
+    changelog = "https://github.com/emilk/eon/blob/${finalAttrs.src.tag}/RELEASES.md";
```

https://github.com/NixOS/nixpkgs/issues/367739
```suggestion
    changelog = "https://github.com/emilk/eon/blob/${finalAttrs.version}/RELEASES.md";
```

---

Maybe before #36773 is resolved, we shouldn't nitpick this too much.  I feel like it's more of a preference thing for right now, and both ways have its flaws.